hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -155,19 +154,16 @@ withinTxExecutionBudget report =
 
 -- | Generate a UTXO representing /commit/ outputs for a given list of `Party`.
 -- NOTE: Uses 'testPolicyId' for the datum.
+-- NOTE: We don't generate empty commits and it is used only at one place so perhaps move it?
```

Comment:
Is this a TODO? NOTE is really a crucial comment to explain why something is done that way and certainly should not be a prompt to change the code.

Reply:
Also.. why don't we want to generate empty commits? Was the test failing with empty commits?

Reply:
This generator is silly to the point we could replace it with hard-coded values because the only place it's used is in a pure generator with a _fixed_ seed! IOW, it's a unit test and not a property-based test which BTW points to a flaw in our Mutation testing strategy.
The reason why `Nothing` is failing is because in the particular case of the test we are using this generator in, actually generating `Nothing` yields a list with a single element which is hard to reorder differently in a meaningful way 😂

Reply:
The `NOTE` should probably be a `FIXME`